script.js

```diff
@@ -0,0 +1,263 @@
+const Keyboard = {
+    elements: {
+        main: null,
+        keysContainer: null,
+        textArea: null,
+        keys: [],
+    },
+
+    properties: {
+        value: "",
+        capsLock: false,
+        shift: false,
+        ctrl: false,
+        language: "english"
+    },
+
+    init(language) {
+        this.elements.textArea = document.createElement("textarea");
+        this.elements.main = document.createElement("div");
+        this.elements.keysContainer = document.createElement("div");
+        this.elements.descriptionContainer = document.createElement("p");
+
+        this.elements.textArea.classList.add("use-keyboard-input");
```

Comment:
please create variables for all classes('keyboard', 'keyboard__keys' and etc.) and all languages('russian', 'russianShift', and etc.)

Reply:
and use  them